payments/schema/mutations.py

```diff
@@ -584,6 +586,70 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteOrderLineMutation()
 
 
+class SendBerthSwitchOfferMutation(graphene.ClientIDMutation):
+    class Input:
+        offers = graphene.List(graphene.NonNull(graphene.ID))
```

Comment:
```suggestion
        offers = graphene.List(graphene.ID, required=True)
```

Reply:
Fixed. ResendOrderMutation had the same thing, fixed that too